packages/firestore/test/unit/local/index_free_query_engine.test.ts

```diff
@@ -349,14 +412,3 @@ function verifyResult(
     'Result count does not match'
   );
 }
-
-function testQueryData(query: Query, hasLimboFreeSnapshot: boolean): QueryData {
-  return new QueryData(
-    query,
-    TEST_TARGET_ID,
-    QueryPurpose.Listen,
-    ListenSequence.INVALID,
-    version(10),
-    hasLimboFreeSnapshot ? version(10) : SnapshotVersion.MIN
-  );
-}
```

Comment:
I don't care super strongly, but as a general rule I hate test boilerplate and so I preferred having the specialized helper.  Was there a reason to remove it?